src/index.tsx

```diff
@@ -452,6 +452,11 @@ export class Modalize<FlatListItem = any, SectionListItem = any> extends React.C
         toValue: destSnapPoint,
         useNativeDriver,
       }).start();
+
+      if (this.props.onPositionChange && this.beginScrollYValue === 0) {
```

Comment:
I have one proposition. Could you create a local state and if the position is the same as the last one, not fired the function again? so we will avoid multiple triggers for the user?

Reply:
This is now done